src/Slices/ServiceInventory/UI/Components/RowActionsMenu/RowActions.tsx

```diff
@@ -285,7 +292,13 @@ export const RowActions: React.FunctionComponent<InstanceActionsProps> = ({
       </MenuContent>
     </Menu>
   );
-
+  console.log(
```

Comment:
console.log

Reply:
I thought I removed that, I guess I did only on other branch